data/autoyast_sle15/tftp.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e -x
+
+HO=`hostname`
+
+echo test >/srv/tftpboot/test.txt
+time tftp $HO -c get test.txt test2.txt || \
```

Comment:
did you check with YaST developers the error in your verification, looks like some dns problem...

Reply:
I think it was just due to mis-configured vpn

Reply:
Not sure if this is nromal indeed. I'd have just written "localhost" or 127.0.0.1 instead of putting hostname in a variable though.

Reply:
I changed that, but may still use that test run to report the bug, if any.